softwarereview_editor.pt.Rmd

```diff
@@ -101,10 +101,18 @@ Cada submissão deve ser revisada por *dois* revisores de pacotes. Embora seja a
 - Após o envio de cada revisão,
   - Escreva um comentário agradecendo ao revisor;
   - Registre a revisão digitando um novo comentário `@ropensci-review-bot submit review <review-url> time <time in hours>`. Por exemplo, para a revisão [https://github.com/ropensci/software-review/issues/329#issuecomment-809783937](https://github.com/ropensci/software-review/issues/329#issuecomment-809783937) o comentário seria `@ropensci-review-bot submit review https://github.com/ropensci/software-review/issues/329#issuecomment-809783937 time 4`.
-- Se o autor deixar de responder, consulte [as políticas de uso](#policies) e/ou envie um _ping_ para os outros editores no canal de discussão do Slack. É importante ressaltar que, se um revisor tiver sido atribuído a um _issue_ fechado, entre em contato com ele ao fechar o _issue_ para explicar a decisão, agradeça-o mais uma vez por seu trabalho e faça uma anotação em nosso banco de dados para atribuí-lo a uma submissão com grandes chances de um processo de revisão de software mais tranquilo da próxima vez (por exemplo, de um autor de pacote que já tenha enviado pacotes para nós anteriormente).
 - Quando as alterações forem feitas, altere a _tag_ de status da revisão para `5/awaiting-reviewer-response` e solicite que os revisores indiquem a aprovação usando o [modelo de aprovação do revisor](#approval2template).
 - Se as pessoas autoras tiverem a intenção de enviar um manuscrito de acompanhamento do tipo [*Applications*](https://besjournals.onlinelibrary.wiley.com/hub/journal/2041210X/features/applicationpapers) no periódico [Methods in Ecology and Evolution](https://besjournals.onlinelibrary.wiley.com/journal/2041210X), indique que o envio do manuscrito pode ser feito após a conclusão da revisão. 
 
+#### Desafios durante a revisão {#challenges-during-review}
+
+- **Se o autor parar de responder** consulte [as políticas](#policies) e/ou envie um ping para os outros editores no canal do Slack para discussão. É importante ressaltar que, se um revisor foi atribuído a um problema fechado, entre em contato com ele ao fechar o problema para explicar a decisão e agradecê-lo mais uma vez pelo trabalho.  Informe os outros editores no canal Slack para que eles sejam considerados para um pacote no futuro com grandes chances de uma revisão de software tranquila (por exemplo, um autor de pacote que já tenha enviado pacotes para nós).
+- **Se um revisor estiver atrasado na revisão ou parar de responder** envie um lembrete após uma semana e novamente após duas semanas. O primeiro lembrete pode ser um `@tag` no GitHub. Depois disso, use e-mail ou outra forma de comunicação direta. Se depois de três semanas você ainda não tiver recebido resposta, determine a melhor maneira de seguir em frente sem eles:
```

Comment:
```suggestion
- **Se um revisor estiver atrasado na revisão ou parar de responder** envie um lembrete após uma semana e novamente após duas semanas. O primeiro lembrete pode ser um `@tag` no GitHub. Depois disso, use o e-mail ou outra forma de comunicação direta. Se depois de três semanas você ainda não tiver recebido resposta, determine a melhor maneira de seguir em frente sem o revisor:
```

>  If after 3 weeks there is still no response, determine how best to move ahead without them:

In this case, "without them" refers to the reviewer, right?

Reply:
Yes.  Perhaps I should change these in English, too, to be clearer.